tutorials/time-series-modeling/Lab_05.md

```diff
@@ -0,0 +1,392 @@
+
+For your project, it is important to understand the characteristics of both your data and what you are trying to measure. Building a time series model to understand cyclical changes in vegetation can provide useful information in understanding crop yield - but if you do not account for issues in the data, you can end up building a time series model  with erroneous results. Many time series modeling tools, such as ARIMA modeling, are not directly applicable in certain settings due to missing data, non-standard collection periods and varying intensity due to atmospheric conditions. In this lab, we will focus on understanding on linear trends and harmonic modeling.  
+
+### Multi-Temporal Data in Earth Engine
+
+Time series data in Earth Engine are represented as a series of images called 'Image Collections'. As a result of the complicating factors in remote sensing discussed earlier, analyzing time series in Earth Engine is unlike time series modeling in traditional methods. From a programming sense, we will join data together to define temporal relationships between collection items and build functions to reduce this time. 
+
+First, some very basic mathematical notation for time series. A time series is an array of the value being measured, sorted chronologically: { $\textbf{p}_{t}  = t_{0} + t_{1}... t_{N}$ }, where each *t* is the given value in the series.
+
+
+### Data Preparation and Preprocessing
+The first step in analysis of time series data is to import data of interest and plot the data around our region of interest, a deciduous forest near Blacksburg, VA.
+
+We begin by loading in the Landsat 8 Collection and provide a point at the region of interest. Additionally, we will create a time field. 
+
+```javascript
+var landsat_8_sr = ee.ImageCollection("LANDSAT/LC08/C01/T1_SR")
```

Comment:
We should use Landsat 8 Collection 2.
Here are the next couple chunks of code updated for L8 C2, please use it to update the code blocks:
[https://code.earthengine.google.com/07bb62d6b88446315076e48a5aa2f043](https://www.google.com/url?q=https://code.earthengine.google.com/07bb62d6b88446315076e48a5aa2f043&sa=D&source=docs&ust=1644270595612570&usg=AOvVaw0t00O-q6JEfrlEoJi5D_W_)

Reply:
Updated all the code chunks